### DISCUSSION TITLE
DBeaver interface tutorials?

### DISCUSSION THREAD

**Discussion by jenyagute74** [dbeaver/dbeaver#11152] [Q&A] [answered]
I am new to SQL/NoSQL stuff and started using DBeaver. I was wondering if there is any video tutorial available for the DBeaver interface (which menu/button do what)?

**Comment by tati-kru** [comment, marked as answer]
Unfortunately, there is not official video tutorials. However, you can find some tutorials on Youtube created by users.

We recommend to use the official DBeaver wiki (https://github.com/dbeaver/dbeaver/wiki)as a knowledge base. It has to be enough at least at the first stage.

**Reply by p91paul** [reply]
This is one of the reddit reposts...

**Reply by tati-kru** [reply]
@p91paul you are right. Anyway, it is really frequent question.
